client/cmd/root.go

```diff
@@ -94,6 +97,8 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&logFile, "log-file", defaultLogFile, "sets Netbird log path. If console is specified the the log will be output to stdout")
 	rootCmd.PersistentFlags().StringVarP(&setupKey, "setup-key", "k", "", "Setup key obtained from the Management Service Dashboard (used to register peer)")
 	rootCmd.PersistentFlags().StringVarP(&preSharedKey, "preshared-key", "p", "", "Sets Wireguard PreSharedKey property. If set, then only peers that have the same key can communicate.")
+	rootCmd.PersistentFlags().StringVar(&wgIface, "wg-iface", iface.WgInterfaceDefault, "WireGuard interface name to use for Netbird client instance")
```

Comment:
let's keep the new flags as part of the up command only. This will prevent confusion too.

Reply:
isn't it also relevant for login?